semgrep/tests/conftest.py

```diff
@@ -0,0 +1,64 @@
+import subprocess
+from pathlib import Path
+from typing import Optional
+from typing import Sequence
+from typing import Union
+
+import pytest
+
+
+TESTS_PATH = Path(__file__).parent
+
+
+def _run_semgrep(
+    config: Optional[Union[str, Path]] = None,
+    *,
+    target_name: str = "basic",
+    stderr: bool = False
+) -> str:
+    config_opts = ["--config", config] if config is not None else []
+
+    return subprocess.check_output(
```

Comment:
At some point in the future, we probably want to be able to assert non-0 return codes

Reply:
This is already possible! Check [`test_rule_parser.py`](https://github.com/returntocorp/semgrep/blob/bence/pytest/semgrep/tests/e2e/test_rule_parser.py).

Sample snippet:

```python
    with pytest.raises(CalledProcessError) as excinfo:
        run_semgrep(f"rules/syntax/{filename}.yaml")
    assert excinfo.value.returncode != 0
```

Reply:
ah right, because we throw a `CalledProcessError`. cool cool.